src/chart/bar/BarView.js

```diff
@@ -424,8 +425,9 @@ function updateStyle(
 
     el.useStyle(zrUtil.defaults(
         {
-            fill: layout.startAngle === layout.endAngle ? 'transparent' : color,
-            opacity: layout.startAngle === layout.endAngle ? 0 : opacity
+            stroke : layout.startAngle === layout.endAngle ? 'none' : stroke,
+            fill: layout.startAngle === layout.endAngle ? 'none' : color,
+            opacity: opacity
```

Comment:
> silent property of the el should be set to true in case it displays again with emphasis style when user is hovering on the area.

Sorry, I was wrong about this. `silent` property is not needed.

But the style of still needs to be eliminated. If I modify the test case and added `emphasis` style. 

```js
series: [{
    type: 'bar',
    data: [0],
    coordinateSystem: 'polar',
    name: 'A',
    roundCap: true,
    color: 'rgba(200, 0, 0, 0.5)',
    itemStyle: {
        borderColor: 'red',
        borderWidth: 1
    },
    emphasis: {
        itemStyle: {
            borderColor: 'blue',
            color: 'red',
            opacity: 1
        }
    }
}]
```

The circle will appear when hovering on the legend.

![image](https://user-images.githubusercontent.com/841551/68274946-59895e00-00a5-11ea-8fb4-b10dc6214987.png)

I think we should also set `fill` and `stroke` to `'none'` of hoverStyle in the code https://github.com/apache/incubator-echarts/blob/master/src/chart/bar/BarView.js#L419

Reply:
sorry about not consider `emphasis` and last week I'm confused by what you say about "silent".  lastweek i have tryed this
```javascript 
    if(layout.startAngle != layout.endAngle){
        graphic.setHoverStyle(el, hoverStyle);    
    }
it seems ok

Reply:
is any one here?

Reply:
Hi @foolzhang  Sorry for missing this.

Just ignore the `silent` suggestion. I've made a mistake. It is for disable the interaction ability of element.

Reply:
> We should always update all to avoid the status leaking from previous update.
reasonable